tlair/src/main/scala/at/forsyte/apalache/tla/lir/oper/TlaFunOper.scala

```diff
@@ -103,27 +105,29 @@ object TlaFunOper {
     *
     * <p>
     * `TlaOperDecl("Fact", List(),
-    *   OperEx(recFunDef,
-    *     OperEx(TlaControlOper.ifThenElse,
-    *       (* n <= 1 *),
-    *       (* 1 *),
-    *       OperEx(Tla.ArithOper.mult,
-    *         NameEx("n"),
-    *         OperEx(TlaFunOper.app,
-    *           OperEx(recFunRef),
-    *           OperEx(TlaArithOper.minus, NameEx(n), ValEx(TlaInt(1)))
-    *         )
-    *       )
-    *     ),
-    *     NameEx("n"),
-    *     TlaIntSet
-    *   )
+    * OperEx(recFunDef,
+    * OperEx(TlaControlOper.ifThenElse,
+    * (* n <= 1 *),
+    * (* 1 *),
+    * OperEx(Tla.ArithOper.mult,
+    * NameEx("n"),
+    * OperEx(TlaFunOper.app,
+    * OperEx(recFunRef),
+    * OperEx(TlaArithOper.minus, NameEx(n), ValEx(TlaInt(1)))
+    * )
+    * )
+    * ),
+    * NameEx("n"),
+    * TlaIntSet
+    * )
```

Comment:
Shall we exclude this reformatting glitch?

Reply:
Yeah, I'd asked about this reformatting on #244 as well, tho this is probably the proper place!

Is the rest of the reformatting in this changeset as intended?

Reply:
I fixed just this block.